doc/rst/technotes/gpu.rst

```diff
@@ -561,6 +561,20 @@ marked with * are covered in our nightly testing configuration.
   * Software:ROCm 4.2*, 4.4, 5.4
 
 
+GPU Support on WSL
```

Comment:
Spell out WSL

Reply:
haha! it was originally and I changed it to this because it loooked long in the menu